src/solver/conv_hip_implicit_gemm_bwd_v4r1_xdlops.cpp

```diff
@@ -852,7 +860,7 @@ ConvSolution ConvHipImplicitGemmBwdDataV4R1Xdlops::GetSolution(
 
     if(!config.IsReallyValid(ctx))
     {
-        MIOPEN_LOG_E("invalid performance parameter");
+        MIOPEN_THROW("invalid performance parameter");
```

Comment:
Please explain how invalid PerformanceConfig can reach this point.

Reply:
I need time to reproduce it.....

Reply:
Do you mean that this is impossible unless a bug in the library?

Reply:
I can't reproduce it now, back to log and watch it

Reply:
?

Reply:
I ever reach this point before,
at last time searching for config,

Reply:
The throw part is another issue and I can’t reproduce, but it’s impossible into this code section, when the issue happened, if I add throw , test pass, but since I can’t find how to reproduce it , I remove the throw part, I don’t know if this answer you ....